scan_name:                   "/rslidar_points" 
scan_queue_size:             2             
scan_modified_name:          "/filtered_cloud" 
scan_modified_queue_size:    2               
scan_candidates_name:        "/velodyne_points"
scan_candidates_queue_size:  2               
pose_measurement_name:       "/object3d_detector/poses"
pose_measurement_queue_size: 2               
clicked_point_name:          "/clicked_point"  
clicked_point_queue_size:    2               
cluster_center_name:         "/cluster_center" 
cluster_center_queue_size:   2               
pipeline_mode:       3    
segmentation:        true
voxel_grid_size:     0.1  
pass_through_min:   -0.4 
pass_through_max:    1.3  
statistical_mean:    50.0 
statistical_std_dev: 0.1  
cluster_tolerance:   0.3  
min_cluster_size:    30   
max_cluster_size:    120  
kf_process_noise:    0.01 
kf_measure_noise:    0.01 
measurement_fail_threshold: 20